runtime/tests/runtime_integration_tests/integration_tests.rs

```diff
@@ -88,6 +88,49 @@ fn test_update() {
     assert_eq!(fw_rev[1], 0xaabbccdd);
 }
 
+///This test will be ignored by default because of its long duration to finish.
+///To run it either remove the ignore attribute(#[ignore]) that is present above the test name
+///(or) pass "--ignored" flag to cargo.
+///Ex: cargo test <test_name> -- --ignored
+#[ignore]
+#[test]
+fn test_stress_update(){
+    let image_options = ImageOptions{
+        app_version: 0xaaabbbbc,
+        ..Default::default()
+    };
+
+    let image =
+        caliptra_builder::build_and_sign_image(&FMC_WITH_UART, &APP_WITH_UART,image_options)
+        .unwrap()
+        .to_bytes()
+        .unwrap();
+
+    let mut model = run_rt_test(None,None,None);
+
+    let stress_num: u32 = 500;
+
+    for _ in 0..stress_num{
+        model
+        .step_until(
+        |m|m.soc_mbox().status().read().mbox_fsm_ps().mbox_idle());
+
+        model
+        .mailbox_execute( u32::from(CommandId::FIRMWARE_LOAD), &image)
```

Comment:
Might be interesting to ping-pong between multiple versions.

Reply:
Thanks for your insight, will update the test to handle multiple versions.